src/actions/order-actions.js

```diff
@@ -558,7 +558,8 @@ export const getPurchaseOrder = (orderId) => async (dispatch, getState) => {
   dispatch(startLoading());
 
   const params = {
-    expand: "tickets, tickets.owner, tickets.owner.member, tickets.ticket_type",
+    expand:
+      "tickets, tickets.owner, tickets.owner.member, tickets.ticket_type, tickets.promo_code",
```

Comment:
please use fields to restrict only to the data needed 
currently is retrieving a lot of unnecessary data